### PR TITLE
Adding OCP version based configmaps for MUO - OSD-6690

### PR DESCRIPTION
**Pull request by ravitri**
Splitting MUO alert silences in different OCP versions as per [OSD-6330](https://issues.redhat.com/browse/OSD-6330) and [OSD-6690](https://issues.redhat.com/browse/OSD-6690).

### CODE REVIEW

**Comment by mrbarge** [comment]
This looks okay to me. Would you mind please adding an OWNERS in the base directory, too? And maybe add yourself, @dofinn , @bmeng and myself to it.

File: deploy/managed-upgrade-operator-config/4.5/10-managed-upgrade-operator-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    scale:
+      timeOut: 30
+    upgradeWindow:
+      delayTrigger: 30
+      timeOut: 120
+    nodeDrain:
+      timeOut: 45
+      expectedNodeDrainTime: 8
+    healthCheck:
+      ignoredCriticals:
```
Comment (wanghaoran1988) [inline deploy/managed-upgrade-operator-config/4.5/10-managed-upgrade-operator-configmap.yaml:33]: Do we need add comments for those alert?

File: deploy/managed-upgrade-operator-config/4.7/config.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+  - key: hive.openshift.io/version-major-minor
+    operator: In
+    values: ["4.7"]
```
Comment (cblecker) [inline deploy/managed-upgrade-operator-config/4.7/config.yaml:6]: Let's move this config up to the parent folder with a NotIn 4.5 or 4.6. We want a "default" config that applies to the current latest version of OSD, plus things like 4.8 nightlies

**Review by cblecker — CHANGES_REQUESTED** [review]
Let a comment. We want a default config.
/hold

File: deploy/managed-upgrade-operator-config/4.5/10-managed-upgrade-operator-configmap.yaml (hunk 1 above)
Comment (ravitri) [reply deploy/managed-upgrade-operator-config/4.5/10-managed-upgrade-operator-configmap.yaml:33]: @wanghaoran1988 - This PR was about the silencing of alerts during controlplane upgrade. The alerts for healthcheck to be reviewed separately as part of [OSD-6691](https://issues.redhat.com/browse/OSD-6691). For now, we are maintaining the healthcheck alerts to be present as it is.

File: deploy/managed-upgrade-operator-config/4.7/config.yaml (hunk 2 above)
Comment (ravitri) [reply deploy/managed-upgrade-operator-config/4.7/config.yaml]: Thanks @cblecker ! Added the changes as requested

**Comment by ravitri** [comment]
@mrbarge - Thanks Matt! Added OWNERS file as requested

**Review by cblecker — APPROVED** [review]

**Pull request merged**